guide/admin/usergroups.md

```diff
@@ -0,0 +1,53 @@
+
+![Administration Link Screenshot](687474703a2f2f692e696d6775722e636f6d2f575a315a6a78392e676966.gif)
+
+Once you are in the Administration Panel, you'll see some navigation tabs flowing down the left hand side, click on **Permissions** to reveal the configuration page to the right hand side.
+
+![Permissions Screenshot](687474703a2f2f692e696d6775722e636f6d2f795834455466302e706e67.png)
+
+User groups are used for all kinds of different purposes, Flarum has a fantastic management system which makes it super-easy to manage user groups, and their associated permissions.
+
+### Create a new User Group
+
+On the configuration page, you'll see some pre-created user groups (if your Flarum installation is a fresh install), these are Admins & Mods, as shown below:
+
+![SS Group bar](687474703a2f2f692e696d6775722e636f6d2f30345a457730472e706e67.png)
+
+Next to those user groups, you'll see **New Group** click that to bring up the **Create Group** prompt.
+
+![SS Group Creation Prompt](687474703a2f2f692e696d6775722e636f6d2f4d4d3039314f4a2e706e67.png)
+
+There are 4 pieces of information Flarum needs from you to create this new user group, for this example we're going to be creating a user group called **Staff Member** but you can name this anything you wish.
+
+### Create Group Form Components
+
+#### Name
+
+ - Singular: is how the name is written should it only apply to one person, in this example our Singular name is **Staff Member**
+ - Plural: is how the name is written should it apply to more than one person, this example would be named **Staff Members**
+ 
+#### Color
+
+Choosing a Color can be done by obtaining the HTML Colour Code of your desired colour, there are plenty of websites you can view which a quick Google search would give you. Enter the colour code you wish to use (starts with a **#**).
+
+#### Icon
+
+The Icon is a small picture that goes at the side of your user avatar in posts to signify that you are a member of this particular user group. To pick your icon, you'll need to visit [this website](http://fontawesome.io/icons/) to pick out your icon, for this example we will be using the ```id-card``` icon. In this field, you'll just need to write "id-card" (without the quotation brackets) and then we're done!
+
+Once you've filled in the form, it should look something like this:
+
+![SS Group Create Filled In](687474703a2f2f692e696d6775722e636f6d2f336863613430662e706e67.png)
+
+Click on **Save Changes** once you've completed the form, which will then create your new user group!
+
+### Delete a User Group
+
+Deleting a user group is very simple, at the top of the configuration page, you'll see all the user groups that you've created, click on the user group that you wish to **delete** which will bring up the user group editing box, bring your mouse to the bottom right of that box over the **Delete group** link, click that and then click **OK** to the warning that comes up. Boom! the user group is now deleted.
```

Comment:
I have re-written the section, including a screenshot to aid visually. Can you let me know what you think of the new section? in testing repo.